CommonMark/Parser/InlineMethods.cs

```diff
@@ -14,11 +14,12 @@ internal static class InlineMethods
         /// Initializes the array of delegates for inline parsing.
         /// </summary>
         /// <returns></returns>
-        internal static Func<Subject, Inline>[] InitializeParsers(CommonMarkSettings settings)
+        internal static Func<Subject, CommonMarkSettings, Inline>[] InitializeParsers(CommonMarkSettings settings)
```

Comment:
hmm... I will look at `PostProcessInlineStack` and see if I can figure a better way...